src/site/stages/build/options.js

```diff
@@ -121,7 +121,9 @@ function deriveHostUrl(options) {
     options.port && options.port !== 80 ? `:${options.port}` : ''
   }`;
 
-  options.domainReplacements = [{ from: 'www\\.va\\.gov', to: options.host }];
+  options.domainReplacements = [
+    { from: 'https://www\\.va\\.gov', to: options.hostUrl },
```

Comment:
This has been broken, but it hasn't ever surfaced, I think, because links in content pages don't typically include www.va.gov. It's now an issue because I'm writing the menu data JSON to the page directly, which is caught in this plugin.